### ISSUE TITLE
No PTR entry for reverse DNS for Privatebin.net/.info

### ISSUE THREAD

**Issue by rugk** [PrivateBin/privatebin.info-pelican#34]
https://stat.ripe.net/app/launchpad/S1_Privatebin.net_C36eC19C30 found two warnings for Privatebin.net:

> WARNING	Nameserver ns1.dssr.ch has an IP address (2001:1620:2057:1:4900::11) without PTR configured.
WARNING	Nameserver ns2.dssr.ch has an IP address (2001:470:b7a1::12) without PTR configured.

See https://www.cloudflare.com/de-de/learning/dns/dns-records/dns-ptr-record/

Ah and also:
> WARNING	SOA 'expire' value (86400) is less than the recommended one (604800).

Just FYI @elrido

**Comment by rugk** [comment]
Same for Privatebin.info as it seems https://stat.ripe.net/app/launchpad/S1_Privatebin.info_C36eC19C30

**Comment by elrido** [comment]
I've added the two pointer records for the nameservers to their respective ipv6.arpa zones and the test above now passes on those points. They currently report connectivity problems to ns2, but I confirmed it is reachable from multiple sites, so maybe they got their subnet temporarily blocked by that site.

I do prefer to keep the SOA record set at 1 day instead of a higher value. I've been bitten by having it set too high (a week) when doing larger migrations of datacenter / provider or such. Not that I plan to do any in the next few years, but times are interesting, so you never know.
